### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by jackchoui** [Maxmentor/Zee5-Latest-Working-API-For-Live-TV#2]
It is not working for me, i found the telegram group but not permitted to ask

**Comment by Maxmentor** [comment]
try this script https://github.com/Maxmentor/Zee5-Latest-Working-API-For-Live-TV/blob/main/zee_api2.php

**Comment by jackchoui** [comment]
yes, i got it to work but know suddenly it doesnt anymore. I get error

Error: Video URL not found in the response.
